docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -40,7 +42,12 @@ Set the systemd restart policy.  The restart-policy must be one of: "no", "on-su
 
 ## Examples
 
-Create and print a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout to stdout.
+### Generate unit files
```

Comment:
The man pages render three and four `#`s the same. I suggest to remove this heading and remove one `#` for all heading below.

If you want to check out the man pages, you can do `make docs` and do a `man docs/build/man/$page.md`